config/config.go

```diff
@@ -304,6 +304,12 @@ func validateAdapters(adapterMap map[string]Adapter, errs configErrors) configEr
 type Metrics struct {
 	Influxdb   InfluxMetrics     `mapstructure:"influxdb"`
 	Prometheus PrometheusMetrics `mapstructure:"prometheus"`
+	Disabled   *DisabledMetrics  `mapstructure:"disabled_metrics"`
```

Comment:
Why is this a pointer?

Reply:
I thought that, if we are going to implelment more metric disabling boolean values inside `config.DisabledMetrics` than just `config.DisabledMetrics.AccountAdapterDetails`, we might as well initialize them all to false (meaning we are not disabling any metrics, all are enable by default), and if the `config.DisabledMetrics` object is not null, then just copy the entire thing with the pointer dereference notation as we do in line 133 of `pbsmetrics/go_metrics.go` shown below. But I realize there are other ways to do this so maybe I'm overthinking things or overcomplicating them using the pointer.
```
 95 func NewBlankMetrics(registry metrics.Registry, exchanges []openrtb_ext.BidderName, disableMetrics *config.DisabledMetrics) *Metrics {
 96     blankMeter := &metrics.NilMeter{}
 97     newMetrics := &Metrics{
 98         MetricsRegistry:            registry,
 99         RequestStatuses:            make(map[RequestType]map[RequestStatus]metrics.Meter),
100 +-- 23 lines: ConnectionCounter:          metrics.NilCounter{},-------------------------------
123         ImpsTypeAudio:  blankMeter,
124         ImpsTypeNative: blankMeter,
125
126         AdapterMetrics:  make(map[openrtb_ext.BidderName]*AdapterMetrics, len(exchanges)),
127         accountMetrics:  make(map[string]*accountMetrics),
128         MetricsDisabled: config.DisabledMetrics{},
129
130         exchanges: exchanges,
131     }
132     if disableMetrics != nil && disableMetrics.AccountAdapterDetails {
133         newMetrics.MetricsDisabled = *disableMetrics //<-- copy the entire object in one line using the pointer notation
134     }
135     for _, a := range exchanges {
136         newMetrics.AdapterMetrics[a] = makeBlankAdapterMetrics()
137     }
138
139     for _, t := range RequestTypes() {
140         newMetrics.RequestStatuses[t] = make(map[RequestStatus]metrics.Meter)
141         for _, s := range RequestStatuses() {
142             newMetrics.RequestStatuses[t][s] = blankMeter
143         }
144     }
145
146     return newMetrics
147 }
pbsmetrics/go_metrics.go
```

Reply:
If you want, I can get rid of the pointer and adjust accordingly